src/Featurizers/StandardScaleWrapperFeaturizer.h

```diff
@@ -0,0 +1,348 @@
+        StandardDeviationAnnotationData   const &        deviation_stats(StandardDeviationEstimator::get_annotation_data(this->get_column_annotations(), _colIndex, Components::StandardDeviationEstimatorName));
+        if (_with_mean) {
+            average = average_stats.Average;
+        }
+        if (_with_std) {
+            deviation = static_cast<std::double_t>(deviation_stats.StandardDeviation);
+        }
+        assert(average_stats.Count == deviation_stats.Count);
+#if (defined __clang__)
+#   pragma clang diagnostic push
+#   pragma clang diagnostic ignored "-Wfloat-equal"
+#   pragma clang diagnostic ignored "-Wdouble-promotion"
+#endif
+        // check if deviation is 0
+        // if it's 0, change it to 1
+        if (deviation == 0) {
+            deviation = 1;
+        }
+
+#if (defined __clang__)
+#   pragma clang diagnostic pop
+#endif
+
+        return typename BaseType::TransformerUniquePtr(new StandardScalerTransformer<InputT, TransformedT>(average, deviation));
+    }
+};
+
+} // namespace Details
+
+/////////////////////////////////////////////////////////////////////////
+///  \class         StandardScaleWrapperEstimator
+///  \brief         Chains up StatisticalMetricsEstimator, StandardDeviationEstimator and StandardScalerTransformerImpl
+///                 to scale input data set so that it's centered around 0 with unit standard deviation
+///
+template <
+    typename InputT,
+    typename TransformedT=std::double_t,
+    size_t MaxNumTrainingItemsV=std::numeric_limits<size_t>::max()
+>
+class StandardScaleWrapperEstimator :
+    public Components::PipelineExecutionEstimatorImpl<
+        Components::StatisticalMetricsEstimator<InputT, MaxNumTrainingItemsV>,
+        Components::StandardDeviationEstimator<InputT, MaxNumTrainingItemsV>,
+        Details::StandardScaleWrapperEstimatorImpl<InputT, TransformedT, MaxNumTrainingItemsV>
+    > {
+public:
+    // ----------------------------------------------------------------------
+    // |
+    // |  Public Types
+    // |
+    // ----------------------------------------------------------------------
+    using BaseType =
+        Components::PipelineExecutionEstimatorImpl<
+            Components::StatisticalMetricsEstimator<InputT, MaxNumTrainingItemsV>,
+            Components::StandardDeviationEstimator<InputT, MaxNumTrainingItemsV>,
+            Details::StandardScaleWrapperEstimatorImpl<InputT, TransformedT, MaxNumTrainingItemsV>
+        >;
+
+    // ----------------------------------------------------------------------
+    // |
+    // |  Public Methods
+    // |
+    // ----------------------------------------------------------------------
+    StandardScaleWrapperEstimator(AnnotationMapsPtr pAllColumnAnnotations, size_t colIndex, bool with_mean, bool with_std);
```

Comment:
Use bit flags rather than bools, as it isn't clear at the call site what "true, true" or "false, true" actuall means.

Reply:
TODO comment added. Will complete bit flag in a separate PR.